README.md

```diff
@@ -1,3 +1,38 @@
 # weather_app
-With this weather app, users can search for the weather forecast for different locations. They will be able to choose whether they want to see the current weather, the hourly forecast for the next 24 hours or a daily forecast for the next 7 days. 
-The project will be done in R Shiny. 
+
+What will you implement first?
+* I will first try to get weather data using the API by https://openweathermap.org.
+
+What is the goal?
+* The goal is to create a weather app where users can get weather forecasts for a location of their choice. They can choose to get information about the current weather, the hourly weather forecast for the next 24 hours, or the daily weather forecast for the next 7 days.  
+
+What do you cut if you’re short on time?
+* I could cut the possibility that users choose between the current weather, an hourly forecast and a daily forecast. Then, the users would always get the current weather and an hourly forecast for the next 24 hours.
+
+What do you extend if you have extra time?
+* I will try to implement a map of the specified location with symbols on the map that are corresponding to the weather forecast for this specific location.   
```

Comment:
I'd recommend the [`leaflet` package](https://rstudio.github.io/leaflet/shiny.html) for this as it integrates well with Shiny.